guides/0116-http_sink.md

```diff
@@ -0,0 +1,47 @@
+# http_sink
+
+* type: consumer
+* input: binary messages
+
+This is a consumer block that takes `data` from incoming `Message` and makes a POST request
```

Comment:
I think Message is a leftover from the original exdoc, just write message (with no backticks).